code/game/objects/effects/adminbus/custom_mines.dm

```diff
@@ -0,0 +1,83 @@
+/obj/effect/custom_mine
+	name = "mine"
+	desc = "Better stay away from that thing"
+	density = FALSE
+	anchored = TRUE
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "uglymine"
+	var/triggered = FALSE
+	var/trigger_on_flying = FALSE
+	var/obj/effect/held_effect
+	var/stay_after_trigger = FALSE
+	var/silent = FALSE
+	var/retrigger_cooldown = 1
+	var/human_only = FALSE
+
+/obj/effect/custom_mine/Initialize(obj/effect/E)
+	. = ..()
+	if(E)
+		held_effect = E
+
+/obj/effect/custom_mine/proc/mineEffect(mob/victim)
+	if(!silent)
+		to_chat(victim, span_danger("*click*"))
+	if(ispath(held_effect, /obj/effect/adminbus/targeted))
+		new held_effect(get_turf(src), victim)
+	else
+		new held_effect(get_turf(src))
```

Comment:
For any reviewing maintainers; This has the unintended effect of making these mines able to create any atom.
I consider it a feature, though I would also understand either a change request to add more sanity or a full-on denial if the typepath does not match an effect.
Please comment as applicable.

Reply:
Mines able to create anything is fine since the only limit is the pre existing code, or mappers.

Reply:
This is fine. Para and I both agree